dev/src/index.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as bun from 'bun';
 import {CallOptions} from 'google-gax';
-import * as through2 from 'through2';
+import {PassThrough, Transform} from 'stream';
```

Comment:
Unused import.

Question, though: `PassThrough` is documented as a trivial `Transform` that passes its input through to its output. By overriding `_transform`, aren't we invalidating the purpose of the refined type? Why not make these all `Transform`?

Reply:
That....  is a fair question 😆